Client.xcodeproj/project.pbxproj

```diff
@@ -6869,7 +6868,6 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				APPLICATION_EXTENSION_API_ONLY = NO;
-				APP_IDENTIFIER_PREFIX = 9G8J6YA743;
```

Comment:
We can remove this from our build settings and use `DEVELOPMENT_TEAM` instead but for the entitlement files we still need to use `AppIdentifierPrefix` as that contains the team plus a trailing period.